exampleConnectionService/src/main/java/com/twilio/voice/examples/connectionservice/VoiceActivity.java

```diff
@@ -0,0 +1,337 @@
+package com.twilio.voice.examples.connectionservice;
```

Comment:
Rename `VoiceActivity.java` to `VoiceConnectionServiceActivity.java` to avoid confusion with `VoiceActivity.java` in the quickstart module.